src/components/Tasks/TaskList.tsx

```diff
@@ -134,20 +157,29 @@ export default function TaskList(props: PropsType) {
 
   return (
     <Grid container spacing={4}>
-
-      <Grid item xs={3} style={{ borderRight: `1px solid ${theme.palette.divider}` }}>
-        <Sidebar tasks={potentialEntries} />
-      </Grid>
-
-      <Grid item xs>
+      <Grid item xs={12} style={{
+        backgroundColor: theme.palette.primary[500], padding: 0, paddingLeft: `calc(25% + ${theme.spacing(2)}px`,
```

Comment:
What's up with the padding? Why such a complex calculation?

Reply:
It's basically to align the search bar with the tasks lists. I actually think I just realized a much better, clearer way to do this. Let me change in next commit